.azure-pipelines.yml

```diff
@@ -118,64 +118,64 @@ stages:
           container_img: mback2k/curl-docker-winbuildenv-msys2-mingw32:ltsc2019
```

Comment:
I guess the MQTT part could be removed from the name here, because that is enabled by default nowadays, right?